rest/models/request.go

```diff
@@ -48,3 +48,30 @@ func QueryParam(key, value string) RequestOption {
 		o.QueryParams.Add(key, value)
 	}
 }
+
+// Headers required to use the Launchpad product.
+const (
+	// HeaderEdgeID is a required Launchpad header. It identifies the Edge User requesting data.
+	HeaderEdgeID = "X-Polygon-Edge-ID"
+	// HeaderEdgeIPAddress is a required Launchpad header. It denotes the originating IP Address of the Edge User requesting data.
+	HeaderEdgeIPAddress = "X-Polygon-Edge-IP-Address "
```

Comment:
just pointing this out: `"X-Polygon-Edge-IP-Address "` -> `"X-Polygon-Edge-IP-Address"`